<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>904</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoMes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: #e4f2f9;</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #17406d; padding: 30px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>265</width>
       <height>58</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>271</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fdfdfd; 
font-size: 48px; 
padding: 0 30px;</string>
     </property>
     <property name="text">
      <string>AutoMes</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>571</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #123153; 
border-radius: 10px;
color: #fdfdfd;
font-size: 16px;
padding: 5px 0 5px 30px;

</string>
     </property>
     <item>
      <property name="text">
       <string>Выберите компанию</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>271</width>
      <height>911</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0f2b49;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>271</width>
       <height>831</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>color: #fbfbfb; font-size: 30px; border: none;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fbfbfb; font-size: 30px; border: none; background: rgba(190, 230, 255, 0.2);</string>
     </property>
     <property name="text">
      <string>ТЕКУЩИЕ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fbfbfb; font-size: 30px; border: none;</string>
     </property>
     <property name="text">
      <string>ОТПРАВЛЕННЫЕ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fbfbfb; font-size: 30px; border: none;</string>
     </property>
     <property name="text">
      <string>ПОЛУЧЕННЫЕ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fbfbfb; font-size: 30px; border: none;</string>
     </property>
     <property name="text">
      <string>СТАТИСТИКА</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>550</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fbfbfb; font-size: 30px; border: none;</string>
     </property>
     <property name="text">
      <string>МОДЕЛИ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>650</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fbfbfb; font-size: 30px; border: none;</string>
     </property>
     <property name="text">
      <string>СТАТИСТИКА</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>750</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fbfbfb; font-size: 30px; border: none;</string>
     </property>
     <property name="text">
      <string>КОМПАНИИ</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>229</y>
      <width>671</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>671</width>
       <height>601</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>671</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #fdfdfd; border-radius: 20px;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #fdfdfd; border-radius: 20px;</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 16px; font-weight: 500; color: #767676; padding-left: 15px;</string>
     </property>
     <property name="text">
      <string>Новое сообщение</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addMsgBtn">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #767676; font-size: 40px; font-weight: 300; text-align: center top; line-height: 30px; padding: 0;</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>scrollArea</zorder>
   <zorder>widget_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
